fix_cbtest_feature_count.patch

```diff
@@ -0,0 +1,319 @@
+diff --git b/test/train-sets/ref/cb_adf_crash2.stderr a/test/train-sets/ref/cb_adf_crash2.stderr
```

Comment:
Adding a patch file so we can fix the unit tests once the finish_examples function is made viable

Reply:
Perhaps put this patch file in the test folder?